nuspec/chocolatey/ChocolateyGUI.install.nuspec

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>chocolateygui.install</id>
+    <version>$version$</version>
+    <title>Chocolatey GUI</title>
+    <authors>Chocolatey</authors>
+    <owners>Chocolatey</owners>
+    <projectUrl>https://github.com/chocolatey/ChocolateyGUI</projectUrl>
+    <projectSourceUrl>https://github.com/chocolatey/ChocolateyGUI</projectSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/ChocolateyGUI/tree/develop/ChocolateyPackage</packageSourceUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/choco/8b6217061a4efe6543622e903996bb8eac012db2/docs/images/Choco_Icon.png</iconUrl>
+    <licenseUrl>https://raw.githubusercontent.com/chocolatey/ChocolateyGUI/develop/LICENSE</licenseUrl>
```

Comment:
just curious, why using the raw url?
Why not link to how it displayed on github, instead of using the download link?

Reply:
@AdmiringWorm good question.  Off the top of me head, I am not sure.  Would need to check the history.  Agree that this should be changed though.